readthedocs/builds/querysets.py

```diff
@@ -27,16 +28,16 @@ class VersionQuerySetBase(models.QuerySet):
     use_for_related_fields = True
 
     def _add_from_user_projects(self, queryset, user, admin=False, member=False):
-        """
-        Add related objects from projects where `user` is an `admin` or a `member`.
-
-        .. note::
-
-           In .org all users are admin and member of a project.
-           This will change with organizations soon.
-        """
-        if user.is_authenticated:
-            projects_pk = user.projects.all().values_list('pk', flat=True)
+        """Add related objects from projects where `user` is an `admin` or a `member`."""
+        if user and user.is_authenticated:
```

Comment:
had to add the user check everywhere 'cause we are passing user as None in several parts from .com :/